apps/src/lib/node/ledger/shell/vote_extensions/eth_events.rs

```diff
@@ -309,16 +309,16 @@ mod test_vote_extensions {
             address: EthAddress([0; 20]),
         };
 
-        oracle.send(event_1.clone()).expect("Test failed");
-        oracle.send(event_3.clone()).expect("Test failed");
+        tokio_test::block_on(oracle.send(event_1.clone())).expect("Test failed");
+        tokio_test::block_on(oracle.send(event_3.clone())).expect("Test failed");
```

Comment:
cool, I didn't know `tokio_test` was a thing

Reply:
It is pretty awesome